README

```diff
@@ -108,6 +108,10 @@ Mandriva:
 	Mandriva has integrated fio into their package repository, so installing
 	on that distro should be as easy as typing ``urpmi fio``.
 
+Arch Linux:
+        Arch Linux package is provided under community.
```

Comment:
Perhaps saying "An Arch Linux package is provided by the community" rather than "Arch Linux package is provided under community" sounds a little smoother?